src/components/EventTypes/CommandFailed.tsx

```diff
@@ -0,0 +1,36 @@
+import { CommandFailedEvent_0_0_2 } from "@texturehq/events";
```

Comment:
It may be useful to have an internal mapping the package to always expose the latest potential for an event type, that maps to the specific version. Would make this more stable going forward for updates